CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 ## 0.9.15 (July TBD, 2019)
+*# Enhancements
+    * Greatly simplified using argparse-based tab completion. The new interface is a complete overhaul that breaks
+    the previous way of specifying completion and choices functions. See header of argparse_completer.py for more
+    information. 
```

Comment:
Linking to argparse_custom.py would be better since I moved all the documentation there. Where should I put the link?

Reply:
I would modify the CHANGELOG to say something like:
* Greatly simplified using argparse-based tab completion. The new interface is a complete overhaul that breaks the previous way of specifying completion and choices functions. See header of [argparse_custom.py](https://github.com/python-cmd2/cmd2/blob/master/cmd2/argparse_custom.py) for more information.

The link won't work until after the PR merges to master, but at least it will give anyone wanting to know more an easy link to more info.